diff --git a/pkgs/swarmauri/pyproject.toml b/pkgs/swarmauri/pyproject.toml
index acc7b9d..411fe0c 100644
--- a/pkgs/swarmauri/pyproject.toml
+++ b/pkgs/swarmauri/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri"
-version = "0.6.2.dev4"
+version = "0.6.2.dev5"
 description = "Namespace package for components, packages, and plugins within the Swarmauri framework."
 license = "Apache-2.0"
 readme = "README.md"
