diff --git a/pkgs/community/swarmauri_parser_textblob/pyproject.toml b/pkgs/community/swarmauri_parser_textblob/pyproject.toml
index 58c7e73..275bf71 100644
--- a/pkgs/community/swarmauri_parser_textblob/pyproject.toml
+++ b/pkgs/community/swarmauri_parser_textblob/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_parser_textblob"
-version = "0.6.2.dev4"
+version = "0.6.2.dev5"
 description = "TextBlob Parser for Swarmauri."
 license = "Apache-2.0"
 readme = "README.md"
