diff --git a/pkgs/community/swarmauri_tool_jupyterexecutenotebookwithparameters/pyproject.toml b/pkgs/community/swarmauri_tool_jupyterexecutenotebookwithparameters/pyproject.toml
index fb11d8b..2ff8585 100644
--- a/pkgs/community/swarmauri_tool_jupyterexecutenotebookwithparameters/pyproject.toml
+++ b/pkgs/community/swarmauri_tool_jupyterexecutenotebookwithparameters/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_tool_jupyterexecutenotebookwithparameters"
-version = "1.0.2.dev3"
+version = "1.0.2.dev4"
 description = "A tool designed to execute parameterized notebooks using papermill, allowing dynamic input and output capture for automated workflows."
 license = "Apache-2.0"
 readme = "README.md"
diff --git a/pkgs/community/swarmauri_tool_jupyterexecutenotebookwithparameters/swarmauri_tool_jupyterexecutenotebookwithparameters/JupyterExecuteNotebookWithParametersTool.py b/pkgs/community/swarmauri_tool_jupyterexecutenotebookwithparameters/swarmauri_tool_jupyterexecutenotebookwithparameters/JupyterExecuteNotebookWithParametersTool.py
index 840dfdd..586a9c7 100644
--- a/pkgs/community/swarmauri_tool_jupyterexecutenotebookwithparameters/swarmauri_tool_jupyterexecutenotebookwithparameters/JupyterExecuteNotebookWithParametersTool.py
+++ b/pkgs/community/swarmauri_tool_jupyterexecutenotebookwithparameters/swarmauri_tool_jupyterexecutenotebookwithparameters/JupyterExecuteNotebookWithParametersTool.py
@@ -17,6 +17,7 @@ the path to the executed notebook output. It inherits from ToolBase and integrat
 with the Swarmauri framework.
 """
 
+
 @ComponentBase.register_type(ToolBase, "JupyterExecuteNotebookWithParametersTool")
 class JupyterExecuteNotebookWithParametersTool(ToolBase):
     """
