diff --git a/pkgs/community/swarmauri_tool_jupyterdisplay/pyproject.toml b/pkgs/community/swarmauri_tool_jupyterdisplay/pyproject.toml
index 79d294b..8908a15 100644
--- a/pkgs/community/swarmauri_tool_jupyterdisplay/pyproject.toml
+++ b/pkgs/community/swarmauri_tool_jupyterdisplay/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_tool_jupyterdisplay"
-version = "0.6.2.dev3"
+version = "0.6.2.dev4"
 description = "A tool designed to display rich media and object representations in a Jupyter Notebook using IPython's display functionality."
 license = "Apache-2.0"
 readme = "README.md"
diff --git a/pkgs/community/swarmauri_tool_jupyterdisplay/swarmauri_tool_jupyterdisplay/JupyterDisplayTool.py b/pkgs/community/swarmauri_tool_jupyterdisplay/swarmauri_tool_jupyterdisplay/JupyterDisplayTool.py
index f89f5a6..c77e7b6 100644
--- a/pkgs/community/swarmauri_tool_jupyterdisplay/swarmauri_tool_jupyterdisplay/JupyterDisplayTool.py
+++ b/pkgs/community/swarmauri_tool_jupyterdisplay/swarmauri_tool_jupyterdisplay/JupyterDisplayTool.py
@@ -15,6 +15,7 @@ from swarmauri_standard.tools.Parameter import Parameter
 from swarmauri_base.tools.ToolBase import ToolBase
 from swarmauri_base.ComponentBase import ComponentBase
 
+
 @ComponentBase.register_type(ToolBase, "JupyterDisplayTool")
 class JupyterDisplayTool(ToolBase):
     """
