diff --git a/pkgs/community/swarmauri_vectorstore_neo4j/pyproject.toml b/pkgs/community/swarmauri_vectorstore_neo4j/pyproject.toml
index 6cf0486..c555f2f 100644
--- a/pkgs/community/swarmauri_vectorstore_neo4j/pyproject.toml
+++ b/pkgs/community/swarmauri_vectorstore_neo4j/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_vectorstore_neo4j"
-version = "0.6.2.dev4"
+version = "0.6.2.dev5"
 description = "Swarmauri Neo4j Vector Store"
 license = "Apache-2.0"
 readme = "README.md"
