diff --git a/pkgs/community/swarmauri_tool_jupyterexecutecell/pyproject.toml b/pkgs/community/swarmauri_tool_jupyterexecutecell/pyproject.toml
index 8de70c4..e1a4742 100644
--- a/pkgs/community/swarmauri_tool_jupyterexecutecell/pyproject.toml
+++ b/pkgs/community/swarmauri_tool_jupyterexecutecell/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_tool_jupyterexecutecell"
-version = "0.6.2.dev3"
+version = "0.6.2.dev4"
 description = "A tool designed to execute a single code cell in a running Jupyter kernel using jupyter_client, capturing its output and errors."
 license = "Apache-2.0"
 readme = "README.md"
diff --git a/pkgs/community/swarmauri_tool_jupyterexecutecell/swarmauri_tool_jupyterexecutecell/JupyterExecuteCellTool.py b/pkgs/community/swarmauri_tool_jupyterexecutecell/swarmauri_tool_jupyterexecutecell/JupyterExecuteCellTool.py
index dd02985..ce1a273 100644
--- a/pkgs/community/swarmauri_tool_jupyterexecutecell/swarmauri_tool_jupyterexecutecell/JupyterExecuteCellTool.py
+++ b/pkgs/community/swarmauri_tool_jupyterexecutecell/swarmauri_tool_jupyterexecutecell/JupyterExecuteCellTool.py
@@ -28,6 +28,7 @@ from swarmauri_base.ComponentBase import ComponentBase
 
 logger = logging.getLogger(__name__)
 
+
 @ComponentBase.register_type(ToolBase, "JupyterExecuteCellTool")
 class JupyterExecuteCellTool(ToolBase):
     """
