diff --git a/pkgs/community/swarmauri_tool_gmail/pyproject.toml b/pkgs/community/swarmauri_tool_gmail/pyproject.toml
index a58bec5..10d84ef 100644
--- a/pkgs/community/swarmauri_tool_gmail/pyproject.toml
+++ b/pkgs/community/swarmauri_tool_gmail/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_tool_gmail"
-version = "0.6.2.dev4"
+version = "0.6.2.dev5"
 description = "example community package"
 license = "Apache-2.0"
 readme = "README.md"
