diff --git a/pkgs/standards/swarmauri_vectorstore_doc2vec/pyproject.toml b/pkgs/standards/swarmauri_vectorstore_doc2vec/pyproject.toml
index ce2c305..3b7aaf6 100644
--- a/pkgs/standards/swarmauri_vectorstore_doc2vec/pyproject.toml
+++ b/pkgs/standards/swarmauri_vectorstore_doc2vec/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_vectorstore_doc2vec"
-version = "0.6.2.dev4"
+version = "0.6.2.dev5"
 description = "A Doc2Vec based Vector Store and Doc2Vec Based Embedding Model."
 license = "Apache-2.0"
 readme = "README.md"
