diff --git a/pkgs/community/swarmauri_tool_jupytergetshellmessage/pyproject.toml b/pkgs/community/swarmauri_tool_jupytergetshellmessage/pyproject.toml
index a17fd1b..c623f1d 100644
--- a/pkgs/community/swarmauri_tool_jupytergetshellmessage/pyproject.toml
+++ b/pkgs/community/swarmauri_tool_jupytergetshellmessage/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_tool_jupytergetshellmessage"
-version = "0.6.2.dev4"
+version = "0.6.2.dev5"
 description = "A tool designed to retrieve shell messages from a running Jupyter kernel using jupyter_client, useful for debugging execution responses."
 license = "Apache-2.0"
 readme = "README.md"
