diff --git a/pkgs/community/swarmauri_vectorstore_mlm/pyproject.toml b/pkgs/community/swarmauri_vectorstore_mlm/pyproject.toml
index 3c3bcca..a566dca 100644
--- a/pkgs/community/swarmauri_vectorstore_mlm/pyproject.toml
+++ b/pkgs/community/swarmauri_vectorstore_mlm/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_vectorstore_mlm"
-version = "0.6.2.dev3"
+version = "0.6.2.dev4"
 description = "Swarmauri MLM Vector Store"
 license = "Apache-2.0"
 readme = "README.md"
diff --git a/pkgs/community/swarmauri_vectorstore_mlm/swarmauri_vectorstore_mlm/MlmVectorStore.py b/pkgs/community/swarmauri_vectorstore_mlm/swarmauri_vectorstore_mlm/MlmVectorStore.py
index d8ff8d2..08148ff 100644
--- a/pkgs/community/swarmauri_vectorstore_mlm/swarmauri_vectorstore_mlm/MlmVectorStore.py
+++ b/pkgs/community/swarmauri_vectorstore_mlm/swarmauri_vectorstore_mlm/MlmVectorStore.py
@@ -12,6 +12,7 @@ from swarmauri_base.vector_stores.VectorStoreSaveLoadMixin import (
 )
 from swarmauri_base.ComponentBase import ComponentBase
 
+
 @ComponentBase.register_type(VectorStoreBase, "MlmVectorStore")
 class MlmVectorStore(
     VectorStoreSaveLoadMixin, VectorStoreRetrieveMixin, VectorStoreBase
