diff --git a/pkgs/standards/swarmauri_parser_keywordextractor/pyproject.toml b/pkgs/standards/swarmauri_parser_keywordextractor/pyproject.toml
index a7a4be8..fc1b7ce 100644
--- a/pkgs/standards/swarmauri_parser_keywordextractor/pyproject.toml
+++ b/pkgs/standards/swarmauri_parser_keywordextractor/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_parser_keywordextractor"
-version = "0.6.2.dev3"
+version = "0.6.2.dev4"
 description = "Keyword Extractor Parser for Swarmauri."
 license = "Apache-2.0"
 readme = "README.md"
diff --git a/pkgs/standards/swarmauri_parser_keywordextractor/swarmauri_parser_keywordextractor/KeywordExtractorParser.py b/pkgs/standards/swarmauri_parser_keywordextractor/swarmauri_parser_keywordextractor/KeywordExtractorParser.py
index 06a56a0..d7f6fd5 100644
--- a/pkgs/standards/swarmauri_parser_keywordextractor/swarmauri_parser_keywordextractor/KeywordExtractorParser.py
+++ b/pkgs/standards/swarmauri_parser_keywordextractor/swarmauri_parser_keywordextractor/KeywordExtractorParser.py
@@ -5,6 +5,7 @@ from swarmauri_standard.documents.Document import Document
 from swarmauri_base.parsers.ParserBase import ParserBase
 from swarmauri_base.ComponentBase import ComponentBase
 
+
 @ComponentBase.register_type(ParserBase, "KeywordExtractorParser")
 class KeywordExtractorParser(ParserBase):
     """
