diff --git a/pkgs/core/pyproject.toml b/pkgs/core/pyproject.toml
index 03825cf..cff7d02 100644
--- a/pkgs/core/pyproject.toml
+++ b/pkgs/core/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri-core"
-version = "0.6.2.dev7"
+version = "0.6.2.dev8"
 description = "This repository includes core interfaces for the Swarmauri framework."
 authors = [
 	{ name = "Jacob Stewart", email = "jacob@swarmauri.com" },
