diff --git a/pkgs/community/swarmauri_vectorstore_persistentchromadb/pyproject.toml b/pkgs/community/swarmauri_vectorstore_persistentchromadb/pyproject.toml
index f4d949c..e6fa751 100644
--- a/pkgs/community/swarmauri_vectorstore_persistentchromadb/pyproject.toml
+++ b/pkgs/community/swarmauri_vectorstore_persistentchromadb/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_vectorstore_persistentchromadb"
-version = "0.6.2.dev3"
+version = "0.6.2.dev4"
 description = "A Persistent ChromaDB based Vector Store"
 license = "Apache-2.0"
 readme = "README.md"
diff --git a/pkgs/community/swarmauri_vectorstore_persistentchromadb/swarmauri_vectorstore_persistentchromadb/PersistentChromaDBVectorStore.py b/pkgs/community/swarmauri_vectorstore_persistentchromadb/swarmauri_vectorstore_persistentchromadb/PersistentChromaDBVectorStore.py
index 52537d1..5c2ab2f 100644
--- a/pkgs/community/swarmauri_vectorstore_persistentchromadb/swarmauri_vectorstore_persistentchromadb/PersistentChromaDBVectorStore.py
+++ b/pkgs/community/swarmauri_vectorstore_persistentchromadb/swarmauri_vectorstore_persistentchromadb/PersistentChromaDBVectorStore.py
@@ -19,6 +19,7 @@ from swarmauri_base.vector_stores.VectorStorePersistentMixin import (
 )
 from swarmauri_base.ComponentBase import ComponentBase
 
+
 @ComponentBase.register_type(VectorStoreBase, "PersistentChromaDBVectorStore")
 class PersistentChromaDBVectorStore(
     VectorStoreSaveLoadMixin,
