diff --git a/pkgs/community/swarmauri_parser_fitzpdf/pyproject.toml b/pkgs/community/swarmauri_parser_fitzpdf/pyproject.toml
index dbc998f..208df23 100644
--- a/pkgs/community/swarmauri_parser_fitzpdf/pyproject.toml
+++ b/pkgs/community/swarmauri_parser_fitzpdf/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_parser_fitzpdf"
-version = "0.6.2.dev4"
+version = "0.6.2.dev5"
 description = "Fitz PDF Parser for Swarmauri."
 license = "Apache-2.0"
 readme = "README.md"
