diff --git a/pkgs/community/swarmauri_vectorstore_qdrant/pyproject.toml b/pkgs/community/swarmauri_vectorstore_qdrant/pyproject.toml
index 194d516..cc8ee17 100644
--- a/pkgs/community/swarmauri_vectorstore_qdrant/pyproject.toml
+++ b/pkgs/community/swarmauri_vectorstore_qdrant/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_vectorstore_qdrant"
-version = "0.6.2.dev3"
+version = "0.6.2.dev4"
 description = "Swarmauri Persistent Qdrant Vector Store"
 license = "Apache-2.0"
 readme = "README.md"
diff --git a/pkgs/community/swarmauri_vectorstore_qdrant/swarmauri_vectorstore_qdrant/CloudQdrantVectorStore.py b/pkgs/community/swarmauri_vectorstore_qdrant/swarmauri_vectorstore_qdrant/CloudQdrantVectorStore.py
index 7a597fd..acb3b62 100644
--- a/pkgs/community/swarmauri_vectorstore_qdrant/swarmauri_vectorstore_qdrant/CloudQdrantVectorStore.py
+++ b/pkgs/community/swarmauri_vectorstore_qdrant/swarmauri_vectorstore_qdrant/CloudQdrantVectorStore.py
@@ -25,6 +25,7 @@ from swarmauri_base.vector_stores.VectorStoreCloudMixin import (
 )
 from swarmauri_base.ComponentBase import ComponentBase
 
+
 @ComponentBase.register_type(VectorStoreBase, "CloudQdrantVectorStore")
 class CloudQdrantVectorStore(
     VectorStoreSaveLoadMixin,
diff --git a/pkgs/community/swarmauri_vectorstore_qdrant/swarmauri_vectorstore_qdrant/PersistentQdrantVectorStore.py b/pkgs/community/swarmauri_vectorstore_qdrant/swarmauri_vectorstore_qdrant/PersistentQdrantVectorStore.py
index ab3440e..761862b 100644
--- a/pkgs/community/swarmauri_vectorstore_qdrant/swarmauri_vectorstore_qdrant/PersistentQdrantVectorStore.py
+++ b/pkgs/community/swarmauri_vectorstore_qdrant/swarmauri_vectorstore_qdrant/PersistentQdrantVectorStore.py
@@ -24,6 +24,7 @@ from swarmauri_base.vector_stores.VectorStorePersistentMixin import (
 )
 from swarmauri_base.ComponentBase import ComponentBase
 
+
 @ComponentBase.register_type(VectorStoreBase, "PersistentQdrantVectorStore")
 class PersistentQdrantVectorStore(
     VectorStoreSaveLoadMixin,
