diff --git a/pkgs/community/swarmauri_llm_leptonai/pyproject.toml b/pkgs/community/swarmauri_llm_leptonai/pyproject.toml
index fb78896..1594d78 100644
--- a/pkgs/community/swarmauri_llm_leptonai/pyproject.toml
+++ b/pkgs/community/swarmauri_llm_leptonai/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_llm_leptonai"
-version = "0.6.2.dev4"
+version = "0.6.2.dev5"
 description = "Swarmauri Lepton AI Model"
 license = "Apache-2.0"
 readme = "README.md"
