diff --git a/pkgs/community/swarmauri_tool_webscraping/pyproject.toml b/pkgs/community/swarmauri_tool_webscraping/pyproject.toml
index d653134..cd8685e 100644
--- a/pkgs/community/swarmauri_tool_webscraping/pyproject.toml
+++ b/pkgs/community/swarmauri_tool_webscraping/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_tool_webscraping"
-version = "0.6.2.dev4"
+version = "0.6.2.dev5"
 description = "Web Scraping Tool for Swarmauri"
 license = "Apache-2.0"
 readme = "README.md"
