diff --git a/pkgs/community/swarmauri_parser_pypdftk/pyproject.toml b/pkgs/community/swarmauri_parser_pypdftk/pyproject.toml
index ec5637d..b0d0f95 100644
--- a/pkgs/community/swarmauri_parser_pypdftk/pyproject.toml
+++ b/pkgs/community/swarmauri_parser_pypdftk/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_parser_pypdftk"
-version = "0.6.2.dev4"
+version = "0.6.2.dev5"
 description = "A parser for extracting text from PDFs using PyPDFTK."
 license = "Apache-2.0"
 readme = "README.md"
