diff --git a/pkgs/community/swarmauri_vectorstore_redis/pyproject.toml b/pkgs/community/swarmauri_vectorstore_redis/pyproject.toml
index 452ae71..4a013ac 100644
--- a/pkgs/community/swarmauri_vectorstore_redis/pyproject.toml
+++ b/pkgs/community/swarmauri_vectorstore_redis/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_vectorstore_redis"
-version = "0.6.2.dev4"
+version = "0.6.2.dev5"
 description = "Swarmauri Redis Vector Store"
 license = "Apache-2.0"
 readme = "README.md"
