diff --git a/pkgs/community/swarmauri_tool_folium/pyproject.toml b/pkgs/community/swarmauri_tool_folium/pyproject.toml
index 841f11d..d3f8088 100644
--- a/pkgs/community/swarmauri_tool_folium/pyproject.toml
+++ b/pkgs/community/swarmauri_tool_folium/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_tool_folium"
-version = "0.6.2.dev4"
+version = "0.6.2.dev5"
 description = "This repository includes an example of a First Class Swarmauri Example."
 license = "Apache-2.0"
 readme = "README.md"
