diff --git a/pkgs/community/swarmauri_tool_jupytervalidatenotebook/pyproject.toml b/pkgs/community/swarmauri_tool_jupytervalidatenotebook/pyproject.toml
index 28b5869..460d807 100644
--- a/pkgs/community/swarmauri_tool_jupytervalidatenotebook/pyproject.toml
+++ b/pkgs/community/swarmauri_tool_jupytervalidatenotebook/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_tool_jupytervalidatenotebook"
-version = "0.6.2.dev4"
+version = "0.6.2.dev5"
 description = "A tool that validates a NotebookNode object against the Jupyter Notebook schema using nbformat, ensuring structural correctness."
 license = "Apache-2.0"
 readme = "README.md"
