diff --git a/pkgs/community/swarmauri_tool_zapierhook/pyproject.toml b/pkgs/community/swarmauri_tool_zapierhook/pyproject.toml
index 5ba161c..523e5aa 100644
--- a/pkgs/community/swarmauri_tool_zapierhook/pyproject.toml
+++ b/pkgs/community/swarmauri_tool_zapierhook/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_tool_zapierhook"
-version = "0.6.2.dev4"
+version = "0.6.2.dev5"
 description = "Zapier Hook Tool"
 license = "Apache-2.0"
 readme = "README.md"
