diff --git a/pkgs/standards/swarmauri_tool_matplotlib/pyproject.toml b/pkgs/standards/swarmauri_tool_matplotlib/pyproject.toml
index 3567e3d..387a8a9 100644
--- a/pkgs/standards/swarmauri_tool_matplotlib/pyproject.toml
+++ b/pkgs/standards/swarmauri_tool_matplotlib/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_tool_matplotlib"
-version = "0.6.2.dev4"
+version = "0.6.2.dev5"
 description = "Matplotlib tool for Swarmauri."
 license = "Apache-2.0"
 readme = "README.md"
