diff --git a/pkgs/community/swarmauri_tool_jupyterexporthtml/pyproject.toml b/pkgs/community/swarmauri_tool_jupyterexporthtml/pyproject.toml
index 2516ebf..0b238d8 100644
--- a/pkgs/community/swarmauri_tool_jupyterexporthtml/pyproject.toml
+++ b/pkgs/community/swarmauri_tool_jupyterexporthtml/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_tool_jupyterexporthtml"
-version = "0.6.2.dev4"
+version = "0.6.2.dev5"
 description = "A tool that exports a Jupyter Notebook to HTML format using nbconvert’s HTMLExporter, enabling web-based presentation of notebooks."
 license = "Apache-2.0"
 readme = "README.md"
