diff --git a/pkgs/community/swarmauri_tool_jupyterexportpython/pyproject.toml b/pkgs/community/swarmauri_tool_jupyterexportpython/pyproject.toml
index 32a6b99..e82b81c 100644
--- a/pkgs/community/swarmauri_tool_jupyterexportpython/pyproject.toml
+++ b/pkgs/community/swarmauri_tool_jupyterexportpython/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "swarmauri_tool_jupyterexportpython"
-version = "0.6.2.dev4"
+version = "0.6.2.dev5"
 description = "A tool that exports a Jupyter Notebook to a Python script using nbconvert’s PythonExporter, facilitating code extraction."
 license = "Apache-2.0"
 readme = "README.md"
